kubectl-rook-ceph.sh

```diff
@@ -49,7 +49,9 @@ function print_usage() {
   echo "  debug <subcommand>...                     : Debug a deployment by scaling it down and creating a debug copy. This is supported for mons and OSDs only"
   echo "    start  <deployment-name>
            [--alternate-image <alternate-image>]    : Start debugging a deployment with an optional container image"
-  echo "  stop <deployment-name>                    : Stop debugging a deployment"
+  echo "    stop <deployment-name>                  : Stop debugging a deployment"
+  echo "  dr <subcommand>"
```

Comment:
I'd suggest we remove `<subcommand>` from all the help commands. It is already obvious enough from the structure of the help.
```suggestion
  echo "  dr"
```

Reply:
removed in a separate commit.